common/src/main/java/org/schemarepo/config/Config.java

```diff
@@ -90,6 +105,9 @@
 
     DEFAULTS.setProperty(DEFAULT_SUBJECT_VALIDATORS,"");
 
+    // Validation defaults. The read strategy class is "Always Fail" unless configured.
+    DEFAULTS.setProperty(VALIDATION_READ_STRATEGY_CLASS, "org.schemarepo.validation.AlwaysFailValidationStrategy");
```

Comment:
Ok, so there might be some Guice trickery involved here... but assuming we could do whatever we wanted with the dependency injection framework, let me ask you this:
* Do you see any use case for overriding the CanReadStrategy with something else? (Of course, there could be non-Avro implementations, but if the user wishes to use Avro, then is there anything else relevant?)
* Do you think it would be desirable to get to a point where the user can simply specify ```schema-repo.validation.type=avro``` in the config (or something along those lines), and be good to go?

I am asking these questions from the perspective of trying to understand/decide what we should strive for, forgetting for a moment the limitations of Guice.

If we can agree to a goal, I might have time to take a stab at bending Guice to our will this coming weekend (: ...

I think the current PR offers a good amount of flexibility, but the average user might be interested in a solution where they can get up and running very easily (say, "I want to use Avro and ensure mutual compatibility against all schema").